qutebrowser/utils/version.py

```diff
@@ -148,13 +148,18 @@ def _git_str_subprocess(gitpath):
     if not os.path.isdir(os.path.join(gitpath, ".git")):
         return None
     try:
-        cid = subprocess.check_output(
-            ['git', 'describe', '--tags', '--dirty', '--always'],
+        commit_hash = subprocess.check_output(
+            ['git',
+             'describe',
+             '--match=NeVeRmAtCh',
+             '--always',
+             '--abbrev=40',
```

Comment:
I don't think we need the `--abbrev=40` - probably fine (and nicer) to show the short hash.